functionality/CLI/team23.go

```
@@ -0,0 +1,28 @@
+package functionality
+
+import (
+	"fmt"
+	//"reflect"
+	"os"
+)
+
+func test(input string) (int){
+	return 0
+}
+
+func READ_url_file(input []string) (int) {
+	filename := os.Args[2]
+
+	contents, err := os.ReadFile(filename)
+    if err != nil {
```

Can we throw an error instead?